### ISSUE TITLE
Computers show in web panel, but they are not mining,

### ISSUE THREAD

**Issue by 1Legend11** [UnamSanctam/SilentCryptoMiner#289]
The computers show in the web panel but not active in my pool and aren't mining in the web panel.

I have them set to mine XMR and ETH and neither are mining,
![Screenshot_156](https://user-images.githubusercontent.com/63114248/149171869-bca973f2-e092-4fef-8cd4-39fc6b327cf4.png)

**Comment by 1Legend11** [comment]
Thanks for the quick response, here are both settings:
![Screenshot_158](https://user-images.githubusercontent.com/63114248/149173303-3a9a3a86-fd67-4151-8a83-cffdb9c0688e.png)
![Screenshot_157](https://user-images.githubusercontent.com/63114248/149173305-a3a2e10c-0164-4e12-b821-02e8b6774f7f.png)

**Comment by 1Legend11** [comment]
Yeah I know ETH wont work bc they aren't 4GB cards, and yes I have SSL enabled in the mining tab

**Comment by 1Legend11** [comment]
2Miners XMR is down for me, I tried

**Comment by 1Legend11** [comment]
MineXMR pool?

**Comment by UnamSanctam** [comment]
I would try another pool then, the two worst pools to silent mine on is MineXMR and SupportXMR.

**Comment by 1Legend11** [comment]
HashCity?

**Comment by UnamSanctam** [comment]
You can try it, the most common ones are 2miner, MoneroOcean and nanopool.

**Comment by 1Legend11** [comment]
I will try nanopool

**Comment by UnamSanctam** [comment]
For nanopool don't put the worker name in the "Worker Name" field since they use their own custom format which is `WalletAddress.WorkerName/WithdrawalPassword` into the "Wallet Address" field.

**Comment by 1Legend11** [comment]
OK, so I should use that format and set a withdraw password?

**Comment by UnamSanctam** [comment]
Yes, the withdrawal password is used to lower the minimum withdrawal amount.

**Comment by 1Legend11** [comment]
OK thanks

**Comment by masterjek** [comment]
I already asked a similar question earlier. I changed 4 different pools of which nanopool. And on the Webpanel, all miners were displayed at a speed of 0 H/s. What I've done. Removed Web panel from hosting. And then I installed it again. And many workers have normal speed. After reinstalling the Web panel, all new workers appear again at a speed of 0 H/s. I reinstall the Web panel on the hosting again and the new workers get a normal speed. So I think the pool fault here is minimal.
